src/main/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/BaseRecalibratorDataflow.java

```diff
@@ -118,7 +121,7 @@ protected void setupPipeline(Pipeline pipeline) {
             BaseRecalibratorDataflowUtils.ensureReferenceIsReadable(pipeline.getOptions(), referencePath);
```

Comment:
Beyond the scope of this branch -- if you think it should be optional, create a separate issue or PR.